packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -143,7 +144,11 @@ const TRANSFORMS = {
       },
       current_operations: obj.current_operations,
       hostname: obj.hostname,
-      iSCSI_name: otherConfig.iscsi_iqn || null,
+      iscsiIqn: versionSatisfies(softwareVersion.product_version, '>=7.5.0')
+        ? obj.iscsi_iqn
+        : otherConfig.iscsi_iqn === undefined
+        ? ''
+        : otherConfig.iscsi_iqn,
```

Comment:
You can (and should) take advantage of the [`??` operator](https://babeljs.io/docs/en/babel-plugin-proposal-nullish-coalescing-operator).